modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -29,6 +31,26 @@ function allHexChars(maybe: string): boolean {
   return maybe.match(/^[0-9a-f]+$/i) !== null;
 }
 
+function arrayEqual(a, b) {
```

Comment:
Add js doc

Reply:
Try to use this. https://www.geeksforgeeks.org/lodash-_-isequal-method/

Reply:
+1